### PR TITLE
VOL-3110: Added Liveliness probe for volta-apps.

### PR DESCRIPTION
**Pull request by kdarapu**
Liveliness is checked using the ONOS_APP active status in ONOS pod.

### CODE REVIEW

File: onos-classic/templates/statefulset.yaml [hunk 1]
```diff
@@ -71,6 +71,8 @@ spec:
         env:
           - name: JAVA_OPTS
             value: -Xmx{{ .Values.heap }}
+          - name: ONOS_APPS
+            value: "org.opencord.sadis,org.opencord.olt,org.opencord.kafka"
```
Comment (Andrea-Campanella) [inline onos-classic/templates/statefulset.yaml:75]: this shoudl probably be empty and allow overwrite. in the `values` file

Comment (kuujo) [reply onos-classic/templates/statefulset.yaml:75]: Yes this is what I did in the original version of this Helm chart IIRC

File: onos-classic/templates/configmap-live-probe.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}live-probe-scripts
```
Comment (Andrea-Campanella) [inline onos-classic/templates/configmap-live-probe.yaml:5]: I would suggest adding a `-` before live, so it's separated.

File: onos-classic/templates/statefulset.yaml [hunk 3]
```diff
@@ -127,5 +132,9 @@ spec:
           configMap:
             name: {{ template "fullname" . }}-probe-scripts
             defaultMode: 0744
+        - name: live-probe-scripts
+          configMap:
+            name: {{ template "fullname" . }}live-probe-scripts
```
Comment (Andrea-Campanella) [inline onos-classic/templates/statefulset.yaml:137]: you'd have to add the `-` also here before live.

**Comment by Andrea-Campanella** [comment]
I'll let @kuujo merge if he is ok with this

**Pull request merged**